hs_modelinstance/forms.py

```diff
@@ -45,30 +49,47 @@ def clean_includes_output(self):
         else:
             return True
 
-#ExecutedBy element forms
-class ExecutedByFormHelper(BaseFormHelper):
-    def __init__(self, allow_edit=True, res_short_id=None, element_id=None, element_name=None,  *args, **kwargs):
 
+# ExecutedBy element forms
+class ExecutedByFormHelper(BaseFormHelper):
+    def __init__(self, allow_edit=True, res_short_id=None, element_id=None, element_name=None, *args, **kwargs):
         # the order in which the model fields are listed for the FieldSet is the order these fields will be displayed
         field_width = 'form-control input-sm'
         layout = Layout(
-                        Field('name', css_class=field_width),
-                 )
+            Field('model_name', css_class=field_width),
+        )
 
         kwargs['element_name_label'] = 'Model Program used for execution'
-        super(ExecutedByFormHelper, self).__init__(allow_edit, res_short_id, element_id, element_name, layout,  *args, **kwargs)
+        super(ExecutedByFormHelper, self).__init__(allow_edit, res_short_id, element_id, element_name, layout, *args,
+                                                   **kwargs)
 
 
 class ExecutedByForm(ModelForm):
     def __init__(self, allow_edit=True, res_short_id=None, element_id=None, *args, **kwargs):
+        # pop owner from kwargs so that it isn't passed into parent classes (below)
+        owner = kwargs.pop('owner')
+
         super(ExecutedByForm, self).__init__(*args, **kwargs)
         self.helper = ExecutedByFormHelper(allow_edit, res_short_id, element_id, element_name='ExecutedBy')
 
+        mp_resource = users.get_resource_list(user=owner, types=['ModelProgramResource'])
+
+        # change above line to this once issue #262 is merged into develop
+        # mp_resource = users.get_resource_list(types=['ModelProgramResource'])
+
+
+        CHOICES = tuple([('Unknown', 'Unknown')] + [(r.short_id, r.title) for r in mp_resource.values()[0]])
```

Comment:
why not make a tuple directly, rather than a list that you just convert inot a tuple?